web/src/features/fireWeather/components/maps/stationAccuracy.test.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  computeAccuracyColors,
+  StationMetrics,
+  neutralColor,
+  noDataColor,
+  tempColorScale,
+  rhColorScale,
+  computePercentageDifference,
+  computeScaleIndex,
+  neutralIndex,
+  differenceToMagnitude
+} from './stationAccuracy'
+
+describe('Station map color accuracy', () => {
+  describe('computeAccuracyColors', () => {
+    it('should return the neutral color code when observed an forecasted are equal', () => {
+      const exactForecast: StationMetrics = {
+        observations: {
+          temperature: 0,
+          relative_humidity: 0
+        },
+        forecasts: {
+          temperature: 0,
+          relative_humidity: 0
+        }
+      }
+      expect(computeAccuracyColors(exactForecast)).toEqual({
+        temperature: neutralColor,
+        relative_humidity: neutralColor
+      })
+    })
+
+    it('should return the no data color code when observed or forecasted is null', () => {
+      const nullForecast: StationMetrics = {
+        observations: null,
+        forecasts: null
+      }
+      expect(computeAccuracyColors(nullForecast)).toEqual({
+        temperature: noDataColor,
+        relative_humidity: noDataColor
+      })
+    })
+
+    it('should return the coolest color when metrics are widely over forecasted', () => {
+      const largeDifference: StationMetrics = {
+        observations: {
+          temperature: 0,
+          relative_humidity: 0
+        },
+        forecasts: {
+          temperature: 300,
+          relative_humidity: 300
+        }
+      }
+      expect(computeAccuracyColors(largeDifference)).toEqual({
```

Comment:
this rears its ugly head again... according to the ticket, if the forecast > observed, then the colour should be warm. But I can't remember if that's the logic that the forecasters want (so I'm not sure whether this implementation is correct or if the ticket description is correct).

Reply:
From the ticket: "warm (orange) colours mean the RH was lower than forecasted, cool colours (green) mean the RH was higher than forecasted"

The screenshot above shows forecasted RH at 20, observed at 26, for Churn Creek, so RH was higher than forecasted and thus green.

Reply:
Just realized the logic implements this but the test describes it wrong, thanks!